docs/marionette.layoutview.md

```diff
@@ -2,6 +2,10 @@
 
 # Marionette.LayoutView
 
+**_DEPRECATED The `LayoutView` has been removed from Marionette 3. See
+the [documentation for View](./marionette.view.md) for the latest documentation
+for managing layouts._**
+
```

Comment:
In which case I'll pull out the CompositeView docs as well. I can review the history to figure out what goes where.